kasa/deviceconfig.py

```diff
@@ -1,10 +1,27 @@
-"""Module for holding connection parameters.
+"""Configuration for connecting directly to a device without discovery.
+
+If you are connecting to a newer KASA or TAPO device you can get the device
+via discovery or connect directly with :class:`DeviceConfig`.
+
+Example:
+    Discovery returns a list of discovered devices:
+
+    >>> from kasa import Discover, Credentials
+    >>> device = await Discover.discover_single(
+    >>>     "127.0.0.1",
+    >>>     credentials=Credentials("myusername", "mypassword"),
+    >>>     discovery_timeout=10
+    >>> )
+    >>> config = device.config # DeviceConfig.to_dict() can be used to store for later
```

Comment:
Instead of just commenting on `to_dict`, it's better to also show it by printing it out how it looks like :-)

Reply:
Done